src/set.js

```diff
@@ -0,0 +1,94 @@
+export default class Set {
+  constructor() {
+    this.length = 0
+    this.elements = []
+  }
+
+  add(element) {
+    this.length === 0
+      ? this.elements.push(element)
+      : this.elements[this.length] = element
+    this.length += 1
+  }
+
+  isEmpty() {
+    return this.length === 0
+  }
+
+  contains(searchTerm) {
+    return this.elements.includes(searchTerm)
+  }
+
+  remove(searchTerm) {
+    const array = this.elements
+    const index = array.indexOf(searchTerm)
+    if (!array.includes(searchTerm)) {
+      return null
+    } else if (index >= 0){
+      array.splice(index, 1)
+      this.length -= 1
+    }
+  }
+
+  forEach(sett, callback) {
+    for (var i in sett){
+      sett[i] = callback(sett[i])
+    }
+  }
+
+  size() {
+    return this.length
+  }
+
+  setUnion(arr2) {
```

Comment:
You could have solved this using reduce also. I only mention that because I finally became comfortable with it this week. Here is my implementation if you are interested: 
```
union( otherSet ) {
    if( !otherSet || !Array.isArray( otherSet ) ) {
      return this.elements
    }
    return otherSet.reduce( ( memo, element ) => {
      if ( !memo.includes( element ) ) {
        memo.push( element )
      }
      return memo
    }, this.elements ).sort( ( element, nextElement ) => {
      return element > nextElement
    })
  }